### ISSUE TITLE
[tslint-immutable] Missing converters

### ISSUE THREAD

**Issue by rafaelss95** [typescript-eslint/tslint-to-eslint-config#1196]
### 💥 Missing Converter
-   `tslint-to-eslint-config` version: 2.9.1
-   `typescript-eslint` version: 7.32.0

#### TSLint Rule Name
All rules can be added.

#### Package Source
TSLint package: https://github.com/jonaskello/tslint-immutable
ESLint package: https://github.com/jonaskello/eslint-plugin-functional

I'd like to work on this, if accepted :)

**Comment by JoshuaKGoldberg** [comment]
Indeed, similar to #1056 this would be a great issue to see resolved!

**Comment by rafaelss95** [comment]
I'll wait the conclusion of #1203 to work on this and #1208.
